src/towncrier/test/test_check.py

```diff
@@ -476,14 +476,28 @@ def test_ignored_files(self, runner):
         """
         When `ignore` is set in config, files with those names are ignored.
         """
-        create_project("pyproject.toml", extra_config='ignore = ["template.jinja"]')
+        create_project(
+            "pyproject.toml",
+            extra_config='ignore = ["template.jinja", "star_wildcard*", "question_wildcard_?", '
+            '"seq_wildcard_[ab]"]',
```

Comment:
Just to keep it simple
```suggestion
            extra_config='ignore = ["template.jinja", "star_wildcard*"]',
```